migrations/sql/V1.29__alter_mine_party_appt.sql

```diff
@@ -0,0 +1,6 @@
+
+ALTER TABLE public.mine_party_appt drop active_ind;
+ALTER TABLE public.mine_party_appt add deleted_ind boolean NOT NULL DEFAULT 'false' ;
+ALTER TABLE public.mine_party_appt DROP CONSTRAINT mine_party_appt_permit_fk;
+ALTER TABLE permit ADD CONSTRAINT mine_permit_guid_unique UNIQUE (permit_guid, mine_guid);
+ALTER TABLE public.mine_party_appt ADD CONSTRAINT mine_party_appt_permit_party_fk FOREIGN KEY (permit_guid,mine_guid) REFERENCES permit(permit_guid,mine_guid);
```

Comment:
I may be interpreting this incorrectly, but I am not fully understanding this foreign key... in the file V1.14__Add_party_appt_and_party_appt_type.sql, the foreign keys look to be all set up.

Reply:
Line 6 is a foreign key pair, so if permit guid and mine guid are both populated, the FK is only valid if that pair exists, not just if both of those values exists separately, the fk in V1.14 only checks that the permit guid exists, (but the permit could belong to a different mine, which wouldn't make sense).

Reply:
Right. Thanks. Now I think I remember what the intent was with this validation. My only concern is that I think this would mean the permit_guid and mine_guid columns in the mine_party_appt table would have 2 foreign keys referencing them? Can this be done? Are we trying to do 2 things with the same column?

Reply:
Clarified in person, let me know if you would like further details! 👍